lib/class-wp-rest-menus-controller.php

```diff
@@ -352,16 +352,18 @@ public function create_item( $request ) {
 			 * If we're going to inform the client that the term already exists,
 			 * give them the identifier for future use.
 			 */
-			$term_id = $term->get_error_data( 'term_exists' );
-			if ( $term_id ) {
-				$existing_term = get_term( $term_id, $this->taxonomy );
-				$term->add_data( $existing_term->term_id, 'term_exists' );
+
+			if ( in_array( 'menu_exists', $term->get_error_codes(), true ) ) {
+				$existing_term = get_term_by( 'name', $prepared_term['menu-name'], $this->taxonomy );
+				$term->add_data( $existing_term->term_id, 'menu_exists' );
 				$term->add_data(
 					array(
 						'status'  => 400,
-						'term_id' => $term_id,
+						'term_id' => $existing_term->term_id,
```

Comment:
```suggestion
						'menu_id' => $existing_term->term_id,
```

Nit: Maybe we should use `menu_id` here.

Reply:
I am trying to keep the response the same as [this](https://github.com/WordPress/wordpress-develop/blob/57ad9f16dff35a879e856566489266472589ea04/src/wp-includes/rest-api/endpoints/class-wp-rest-terms-controller.php#L460-L470).